__tests__/app.test.js

```diff
@@ -125,4 +126,96 @@ describe("/api/articles", () => {
         
         })
     })
+})
+
+
+describe("/api/articles/:article_id/comments", () => {
+    test("GET 200: Responds with a 200 OK status", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+    })
+
+    test("GET 200: Responds with a list of the correct length", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then(({body})=> {
+           const {comments} = body
+           expect(comments.length).toBe(11)
+
+        })
+    })
+
+    test("GET 200: Responds with a list of all comments from a particular article", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then(({body})=> {
+           const {comments} = body
+
+           comments.forEach((comment)=>{
+            expect(comment.article_id).toBe(1)
+           })
+
+        })
+    })
+
+    test("GET 200: Response contains all of the correct object keys", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then(({body})=> {
+           const {comments} = body
+
+           comments.forEach((comment)=>{
+            expect(comment).toMatchObject({
+                comment_id: expect.any(Number),
+                body: expect.any(String),
+                article_id: expect.any(Number),
+                author: expect.any(String),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+            })
```

Comment:
You could have combined the above four tests into the one, as they are all asserting against the response which is the one grouped behaviour we expect to get back here from this GET request.
Potentially you could include the order by test as well if you wanted to. 
Up to you though really, just something to think about how you want to move forward. The key is to make sure all tests are red first, then green and don't auto pass.